### ISSUE TITLE
BuildQualityChecks: runTitle with $(System.PullRequest.TargetBranch) fails task in Azure DevOps

### ISSUE THREAD

**Issue by jason-ha** [MicrosoftPremier/VstsExtensions#106]
As we have runs that may be compared against several branches, having branch name in the title is useful. However this will produce a task error per `/` in the string.

Example: `runTitle: Check-Previous-Lines-Coverage on $(System.PullRequest.TargetBranch) with 0.5% variance`
 using v7.3.0 results in:
```
2020-07-08T05:57:15.4290812Z ##[error]Unable to process command '##vso[task.addattachment type=Distributedtask.Core.Summary;name=Check-Previous-Lines-Coverage on refs/heads/master with 0.5% variance;]d:\a\1\s\BuildQualityChecksSummary.md' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
2020-07-08T05:57:15.4302456Z ##[error]Name contains invalid characters. (", <, >, |,  , , , , , , , , , 	, 
, , , 
, , , , , , , , , , , , , , , , , , , :, *, ?, \, /)
```

If BuildQualityChecks cannot handle / with some substitution or such, then those limitations should be documented.

**Comment by ReneSchumacher** [comment]
Hi @jason-ha,

thank you for reporting this and sorry for the inconvenience this might have caused. To be honest, we just didn't think about character restrictions while implementing our last change to the summary section title, otherwise we would have documented them. I'll make sure this is fixed in the next version of the task that should be available soon.

René

**Comment by ReneSchumacher** [comment]
Hi again,

we fixed this issue in v7.4.0 and added information about the restrictions to the docs. Thanks again for reporting this and sorry for the inconveniences this might have caused.

Happy building,
René
